templates/author.twig

```diff
@@ -1,35 +1,25 @@
 {% extends "base.twig" %}
 
 {% block content %}
-
-	<div class="clearfix"></div>
-	<div class="skewed-overlay"></div>
-
-	<header class="page-header">
-		<div class="container">
+	<header class="page-header container">
+		{% if ( author.avatar ) %}
+			<figure class="author-image" aria-label="{{ __( 'Picture of ' ~ author.name , 'planet4-master-theme' ) }}">
+				<img itemprop="image" class="author-pic"
+					 src="{{ fn('get_avatar_url', author.id, {'size' : 300}) }}"
+					 alt="{{ author.name }}">
+			</figure>
+		{% endif %}
+		<div class="author-details">
 			<h1 class="page-header-title">{{ author.name }}</h1>
-			<div class="row">
-				<div class="col-lg-7 col-md-8 col-sm-12">
-					{% if ( author.avatar ) %}
-						<figure class="author-image" aria-label="{{ __( 'Picture of ' ~ author.name , 'planet4-master-theme' ) }}">
-							<img itemprop="image" class="author-pic"
-									src="{{ fn('get_avatar_url', author.id, {'size' : 300}) }}"
-									alt="{{ author.name }}">
-						</figure>
-					{% endif %}
-					{% if ( author.description ) %}
-						<p class="d-md-block author-bio" aria-label="{{ __( 'Author bio', 'planet4-master-theme' ) }}">{{ fn('wpautop', fn('the_author_meta', 'description', author.id))|e('wp_kses_post')|raw }}</p>
-					{% endif %}
-				</div>
-				<div class="col-lg-5 col-md-4 col-sm-12 mt-3 mt-md-0">
-					{% include "blocks/share_buttons.twig" with {social:author_share_buttons} %}
-				</div>
-			</div>
+			{% if ( author.description ) %}
+				<p class="d-md-block author-bio" aria-label="{{ __( 'Author bio', 'planet4-master-theme' ) }}">{{ fn('wpautop', fn('the_author_meta', 'description', author.id))|e('wp_kses_post')|raw }}</p>
+			{% endif %}
 		</div>
 	</header>
 
 	<div class="page-content container">
-		<h3>{{ __( 'Posts by ', 'planet4-master-theme' ) }} {{ author.name }}</h3>
+		<div class="wp-block-separator is-style-wide"></div>
+		<h2>{{ __( 'Posts by ', 'planet4-master-theme' ) }} {{ author.name }}</h2>
```

Comment:
I made this an `h2` to increase the margin bottom as requested by Houssam. It also semantically makes more sense than `h3`.